package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
@joscha I updated the main `lint` command such that it will properly fail on `.storybook` files, so this just won't match them on git commits. It will still fail in CI if there are lint errors.